File: introduction/ITP1_4_D-Min, Max and Sum.c
#include<stdio.h>
#include<math.h>

int main(void) {
  int n, num;
  long minnum = 1000000, maxnum = -1000000, sum = 0;
  scanf("%d", &n);
  while(n--) {
    scanf("%d", &num);
    minnum = fmin(minnum, num);
    maxnum = fmax(maxnum, num);
    sum += num;
  }
  printf("%d %d %ld\n", minnum, maxnum, sum);
  return 0;
}

File: introduction/ITP1_4_D-Min, Max and Sum.java
import java.util.*;
import java.lang.Math;

public class Main {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);

    int n = Integer.parseInt(sc.nextLine());
    long minnum = 1000000, maxnum = -1000000, sum = 0;

    String[] line = sc.nextLine().split(" ");
    for(int i = 0; i < n; i++) {
      Long num = Long.parseLong(line[i]);
      minnum = Math.min(minnum, num);
      maxnum = Math.max(maxnum, num);
      sum += num;
    }

    System.out.println(minnum + " " + maxnum + " " + sum);
  }
}
